bundles/AdminBundle/Resources/public/js/pimcore/document/editables/renderlet.js

```diff
@@ -11,10 +11,10 @@
  * @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
-pimcore.registerNS("pimcore.document.editables.renderlet");
+pimcore.registerNS('pimcore.document.editables.renderlet');
 pimcore.document.editables.renderlet = Class.create(pimcore.document.editable, {
 
-    defaultHeight: 100,
+    defaultHeight: 150,
```

Comment:
```suggestion
    defaultHeight: 100,
```
We should leave the default height as it was (same as snippet).

Reply:
Okay :)